pkgs/development/python-modules/pyexcel-xls/default.nix

```
@@ -30,6 +32,10 @@ buildPythonPackage rec {
     mock
   ];
 
+  postPatch = ''
+    sed -i 's/xlrd<2/xlrd<3/g' setup.py
+  '';
+
   checkPhase = "nosetests";
```

```suggestion
  checkPhase = "nosetests --exclude test_issue_151";
```

Should probably work instead of the patch.

https://nose.readthedocs.io/en/latest/man.html